integreat_cms/cms/templates/pages/page_form.html

```diff
@@ -123,24 +129,35 @@ <h1 class="heading">
                                         <i data-feather="x"></i>
                                     </span>
                                 {% endif %}
+                                <span>
+                                    <span class="fp fp-rounded fp-{{ other_language.primary_country_code }}"></span>
+                                    {% if other_language.secondary_country_code %}
+                                        <span class="fp fp-rounded fp-{{ other_language.secondary_country_code }}"></span>
+                                    {% endif %}
+                                </span>
                                 {{ other_language.translated_name }}
```

Comment:
```suggestion
                                {{ other_language.translated_name }}
                                <span class="fp fp-rounded fp-{{ other_language.primary_country_code }}"></span>
                                {% if other_language.secondary_country_code %}
                                    <span class="fp fp-rounded fp-{{ other_language.secondary_country_code }}"></span>
                                {% endif %}
```
I think having the flags after the language name looks better.